content/docs/command-reference/plots/index.md

```diff
@@ -115,29 +118,30 @@ There are two important fields that DVC adds to the plot data:
   distinguish between different versions when using the `dvc plots diff`
   command.
 
-DVC applies the same logic to all CSV/TSV files, but first transforms the data
-into JSON. DVC uses column names from a header for JSON conversion into fields.
+DVC applies the same logic to all CSV/TSV metric files, but first converts the
+data into JSON format, using column names from the table header (first row) as
+the field names.
```

Comment:
Is this note on the way DVC works internally necessary for the cmd ref? I.e. does knowing this help the user in any way e.g. when they're creating a custom template @pared? Thanks

Reply:
might be relevant if it is always a requirement to have header for DVC to work properly ... I agree that we should rephrase it though, it's not clear where this note comes from now

Reply:
Rephrased.